packages/neon/neon/lib/src/bloc/result.dart

```diff
@@ -67,12 +123,29 @@ class Result<T> {
         isCached: isCached ?? this.isCached,
       );
 
+  /// Returns whether this snapshot contains a non-null [error] value.
+  ///
+  /// This is always true if the asynchronous computation's last result was
+  /// failure.
```

Comment:
I don't think this is helpful and somewhat misleading because there are also other reasons why it might be true.